Update {{AvailableInWorkers}} macros usage for Service Worker API - part I (#33044)
mdn/content: files/en-us/web/api/serviceworker/index.md

Service workers allow static import of [ECMAScript modules](/en-US/docs/Web/JavaScript/Guide/Modules), if supported, using [`import`](/en-US/docs/Web/JavaScript/Reference/Statements/import).
Dynamic import is disallowed by the specification — calling [`import()`](/en-US/docs/Web/JavaScript/Reference/Operators/import) will throw.

> Currently the `ServiceWorker` object is not exposed to {{domxref("DedicatedWorkerGlobalScope")}} and {{domxref("SharedWorkerGlobalScope")}} for Chrome and Firefox.

Reviewer: FWIW I am not sure about these words.
1. I very much doubt the `ServiceWorker` will or ever can be exposed on anything other than [ServiceWorkerGlobalScope](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerGlobalScope).
2. Avoid specifying browser compatibility version in the body - "for Chrome and Firefox.". My first though is that presumably this means that Safari does support this in the other scopes. But either way, if there is an expectation that this should be in those scopes then that should be in the browser compatibility data, not here.

Assuming I am correct, this text might be better as. 

```suggestion
`ServiceWorker` is only exposed in {{domxref("ServiceWorkerGlobalScope")}} (not {{domxref("DedicatedWorkerGlobalScope")}} or {{domxref("SharedWorkerGlobalScope")}}).
```

or "not dedicated or shared workers".

Reviewer: OK, just read your description https://github.com/mdn/content/pull/33044#issue-2238012419 properly 

So in this case I still hold that you need to capture the compatibility information in browser compatibility. The text would then be "something like".

```suggestion
Service workers can currently only be registered in the Window scope, because the `ServiceWorker` object is not exposed to {{domxref("DedicatedWorkerGlobalScope")}} and {{domxref("SharedWorkerGlobalScope")}} in some or all browsers. Check [browser compatibility]() for information.
```

Reviewer: You added a comment below, but this one still exists.

Service workers allow static import of [ECMAScript modules](/en-US/docs/Web/JavaScript/Guide/Modules), if supported, using [`import`](/en-US/docs/Web/JavaScript/Reference/Statements/import).
Dynamic import is disallowed by the specification — calling [`import()`](/en-US/docs/Web/JavaScript/Reference/Operators/import) will throw.

Service workers can only be registered in the Window scope in some or all browsers, because the `ServiceWorker` object is not exposed to {{domxref("DedicatedWorkerGlobalScope")}} and {{domxref("SharedWorkerGlobalScope")}}.

> Check the [browser compatibility](#browser_compatibility) for information.

Reviewer: Again, would be good to add this to the compat data.
